mlir/lib/Dialect/MIOpen/Transforms/ThreadwiseGemmLowering.cpp

```diff
@@ -198,12 +201,12 @@ struct XdlopsGemmV2RewritePattern : public OpConversionPattern<XdlopsGemmV2Op> {
 
     bool IsKReduction = (num_output_blks == 1) && (num_input_blks > 1);
 
-    Value bufferA = adaptor.bufferA();
-    Value bufferB = adaptor.bufferB();
-    auto bufferAType = adaptor.bufferA().getType().cast<MemRefType>();
-    auto bufferBType = adaptor.bufferB().getType().cast<MemRefType>();
-    Type bufferAElementType = bufferAType.getElementType();
-    Type bufferBElementType = bufferBType.getElementType();
+    Value matrixA = adaptor.matrixA();
```

Comment:
I see things seem to have been renamed, but the one place I didn't see a renaming is in the names of the operation arguments themselves?

Reply:
If I understand your question right, there used to be a `matrixA` argument, and whose name I reused (and replaced `bufferA`). So it is for `matrixB` and `bufferB`. You can see the updated diff on `MIOpenOps.td` that has those renames.

Reply:
`MIOpenOps.td` doesn't rename the arguments at all.

What I was trying to work out: was it the case that, before this, "buffer" meant the LDS memref and "matrix" meant the in-register stuff and now you've switched it? It feels like you've switched it

Reply:
See [here](https://github.com/ROCmSoftwarePlatform/llvm-project-mlir/pull/711/files#r940651714) for the rename.

Previously, matrix was lds and buffer is register. Now, matrix is register (and lds is no longer needed in new xdlops because blockwisegemm did the copy already).